.copr/Makefile

```diff
@@ -0,0 +1,38 @@
+
+
+SELF=$(lastword $(MAKEFILE_LIST))
+ROOT_DIR=$(abspath $(dir $(SELF))/..)
+SKIP_DEPS=
+
+outdir:=/var/tmp/copr-tmp-outdir
+spec:=extras/python-sambacc.spec
+
+.PHONY: srpm
+srpm: sys_deps py_deps
+	mkdir -p $(outdir)
+	git fetch --tags
+	SAMBACC_SRPM_ONLY=yes \
+		SAMBACC_BUILD_DIR=$(ROOT_DIR) \
+		SAMBACC_DIST_PREFIX=$(outdir)/.dist \
+		SAMBACC_DISTNAME=copr \
+		SAMBACC_BUILD_TASKS="task_py_build task_rpm_build" \
+		 ./tests/container/build.sh
+	cp $(outdir)/.dist/copr/SRPMS/*.rpm  $(outdir)
```

Comment:
Why do we copy _*.rpm_ to $outdir?

Reply:
The build.sh puts results under a "distdir" but the copr invocation of make passes and outdir variable to put the sprm in, it was simpler than redoing chunks of build.sh.

Reply:
Is it like COPR is expecting an `$outdir` with SRPM?

Reply:
yes

Reply:
https://docs.pagure.org/copr.copr/user_documentation.html#scm   scroll down to `make srpm`

Reply:
Now I know how `dnf` appeared in Makefile 😃